src/target.cpp

```diff
@@ -394,6 +397,8 @@ const char *get_target_os_name(Os os_type) {
             return "freestanding";
         case OsZen:
             return "zen";
+		case OsUefi:
+			return "uefi";
```

Comment:
No tabs: 4 spaces please

Reply:
Should have been taken care of by my editor.  Sorry.